layouts/partials/footer.html

```diff
@@ -5,7 +5,7 @@
         <div class="col-md-4 col-sm-6">
             <h4>{{ i18n "aboutUs" }}</h4>
 
-            {{ .Site.Params.about_us | markdownify }}
+            {{ .Site.Params.about_us | safeHTML }}
```

Comment:
How about `{{ .Site.Params.about_us | markdownify | safeHTML }}` instead? That way we can mix Markdown and HTML in the `about_us` key in `config.toml`